best_line_benchmark.cpp

```diff
@@ -1,21 +1,18 @@
 #include <benchmark/benchmark.h>
 
 #include <ctime>
+#include <random>
 
 #include "best_line.h"
 
 // Function to populate a random graph of rational numbers with n points
 std::vector<Point> PopulateRandomGraph(std::vector<Point> &graph, int n) {
-  double lower_bound = -20000;
-  double upper_bound = 2000;
+  std::mt19937 generator(static_cast<unsigned int>(std::time(NULL)));
```

Comment:
there is also [std::random_device](https://en.cppreference.com/w/cpp/numeric/random/random_device)